cg/services/delivery_message/utils.py

```diff
@@ -75,3 +75,7 @@ def get_case_app_tag(case: Case) -> str:
 
 def get_sample_app_tag(sample: Sample) -> str:
     return sample.application_version.application.tag
+
+
+def is_matching_order(cases: list[Case]) -> bool:
+    return all([case.latest_order == cases[0].latest_order for case in cases])
```

Comment:
I like it! Since the delivery message functionality will be available in the old view, the users can potentially try to generate a delivery message for cases belonging to different orders!